source/math/IntSize3D.ooc

```diff
@@ -32,6 +32,7 @@ IntSize3D: cover {
 	minimum: func (ceiling: This) -> This { This new(Int minimum~two(this width, ceiling width), Int minimum~two(this height, ceiling height), Int minimum~two(this depth, ceiling depth)) }
 	maximum: func (floor: This) -> This { This new(Int maximum~two(this width, floor width), Int maximum~two(this height, floor height), Int maximum~two(this depth, floor depth)) }
 	clamp: func (floor, ceiling: This) -> This { This new(this width clamp(floor width, ceiling width), this height clamp(floor height, ceiling height), this depth clamp(floor depth, ceiling depth)) }
+	//FIXME: Modulo operator does not work for negative numbers
```

Comment:
Yes, but `Int modulo` in `math.ooc` should do that. For some reason it doesn't seem to work for me...